src/js/api/config.js

```diff
@@ -96,6 +96,9 @@ const Config = function(options, persisted) {
     config.mute = !!config.mute;
     config.language = language;
     config.intl = intl;
+    if (__HEADLESS__) {
+        config.controls = false;
```

Comment:
Why not load controls in headless?

Reply:
seriously?

Reply:
this seems to create a disconnect between the Headless player and the SDK, since the SDK will have controls that can be toggled. Is it safe to have this disconnect ?  Is there any functionality i.e. pings that rely on the controls state ?

Reply:
Good point @karimMourra,

The reason this is set to false here is because setting it to true would cause the player here to try to load jwplayer.controls.js. If we want the SDK to use the headless player model, and honor config.controls (which becomes model.attribute.controls) then I can remove this change, and instead add `__HEADLESS__` checks around all places here in the headless player where we act on the controls bool. Let me see how involved that would be...

I was thinking you don't really need the JS player to track controls state. You could manage that entirely in the SDK. This attribute has no influence at all on the provider, it's only used in the view and possibly in parts of **ads plugins** - that's where we would need to be careful.

Reply:
Funny thing is no providers actually implement setControls... so it doesn't really influence providers in the web player. The only reason you would is if you want to controls to toggle your AVPlayer controls, but that's not how we plan to use it.